src/deserialize.rs

```diff
@@ -70,7 +70,7 @@ impl<'de> serde::Deserializer<'de> for RichTerm {
             Term::Num(v) => visitor.visit_f64(v),
             Term::Str(v) => visitor.visit_string(v),
             Term::Enum(v) => visitor.visit_enum(EnumDeserializer {
-                variant: v.label,
+                variant: v.into(),
```

Comment:
Why not replacing this with `v.label()` ? I think it's more explicit than `into()`, which depends on the target type expected.

Reply:
I added a method in Ident `into_label`, to make conversions clearer than using plain `into`